frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js

```diff
@@ -76,68 +97,112 @@ export default function NextStepsRepeater({
-                className="sr-only"
+                htmlFor={`${stepType}-next-step-${index + 1}`}
               >
-                Next step
-                {' '}
-                { index + 1 }
+                {`Step ${index + 1}`}
+                <span className="smart-hub--form-required font-family-sans font-ui-xs text-secondary-dark">
+                  {' '}
+                  *
+                </span>
               </Label>
-              <Textarea
-                key={item.key}
-                id={`${name === 'specialistNextSteps' ? 'specialist' : 'recipient'}-next-step-${index + 1}`}
-                className="height-10 minh-5 smart-hub--text-area__resize-vertical"
-                name={`${name}[${index}].note`}
-                type="text"
-                defaultValue={item.note}
-                inputRef={register({ required: 'Enter a next step' })}
-                onBlur={({ target: { value } }) => validateOnBlur(value, index)}
-                data-testid={`${name === 'specialistNextSteps' ? 'specialist' : 'recipient'}NextSteps-input`}
-                style={{ height: !heights[index] ? `${DEFAULT_STEP_HEIGHT}px` : heights[index] }}
-                onChange={(e) => onStepTextChanged(e, index)}
-              />
-              { canDelete ? (
-                <Button
-                  className="margin-top-0"
-                  unstyled
-                  type="button"
-                  aria-label={`remove ${ariaName} ${index + 1}`}
-                  onClick={() => onRemoveStep(index)}
-                >
-                  <FontAwesomeIcon className="margin-x-1" color="#000" icon={faTrash} />
-                  <span className="sr-only">
-                    remove step
-                    {' '}
-                    { index + 1 }
-                  </span>
-                </Button>
-              ) : null}
-            </div>
-          </FormGroup>
+              {blurStepValidations[index] || (errors[name]
+                && errors[name][index] && errors[name][index].note)
+                ? <ErrorMessage>Enter a next step</ErrorMessage>
+                : null}
+              <div
+                key={`${stepType}-next-step-flex-step-${index + 1}`}
+                className={`display-flex ${blurStepValidations[index]
+                  || (errors[name] && errors[name][index]
+                    && errors[name][index].note) ? 'blank-next-step' : ''}`}
+              >
+                <Textarea
+                  key={item.key}
+                  id={`${stepType}-next-step-${index + 1}`}
+                  className="height-10 minh-5 smart-hub--text-area__resize-vertical"
+                  name={`${name}[${index}].note`}
+                  type="text"
+                  defaultValue={item.note}
+                  inputRef={register({ required: 'Enter a next step' })}
+                  onBlur={({ target: { value } }) => validateStepOnBlur(value, index)}
```

Comment:
Maybe you already did, but I would check this validation message against the mock up as well.